pkg/config/controller_config.go

```diff
@@ -27,13 +27,15 @@ const (
 	flagBackendSecurityGroup                         = "backend-security-group"
 	flagEnableEndpointSlices                         = "enable-endpoint-slices"
 	flagDisableRestrictedSGRules                     = "disable-restricted-sg-rules"
+	flagEnableClusterAwareBindings                   = "enable-cluster-aware-bindings"
```

Comment:
The feature is enabled by a flag on the controller, but it could also be on the binding resource itself.